packages/vite/src/node/plugins/css.ts

```diff
@@ -590,6 +591,25 @@ function getCssResolversKeys(
   return Object.keys(resolvers) as unknown as Array<keyof CSSAtImportResolvers>
 }
 
+function isCssModule(
+  modulesOptions: CSSOptions['modules'],
+  id: string
+): boolean {
+  if (modulesOptions && modulesOptions.isCssModule) {
+    const { isCssModule } = modulesOptions
+    if (typeof isCssModule === 'function') {
+      return isCssModule(id)
+    } else if (
+      typeof isCssModule === 'string' ||
+      isCssModule instanceof RegExp
+    ) {
+      const regexp = new RegExp(isCssModule)
```

Comment:
**question:** Is it allowed to wrap a `RegExp` with a `RegExp`?

Reply:
As of ES5, this can also be another RegExp object or literal .
[MDN RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/RegExp)